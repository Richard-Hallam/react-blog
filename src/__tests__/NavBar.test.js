import React from "react";
import { render, screen } from '@testing-library/react';
import Navbar from "../Components/NavBar";


describe("Navbar", () => {
    const { asFragment } = render(<Navbar />);

    it("renders correctly", ()=>{
        expect(asFragment()).toMatchSnapshot();
    });
});