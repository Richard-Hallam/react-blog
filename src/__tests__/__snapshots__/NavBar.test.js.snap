// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<DocumentFragment>
  <div>
    <h2
      class="nav-header"
    >
      Richard Hallam
    </h2>
    <ul
      class="link-bar"
    >
      <li
        class="link-bar-link"
      >
        Projects
      </li>
      <li
        class="link-bar-link"
      >
        Github
      </li>
      <li
        class="link-bar-link"
      >
        Linkedin
      </li>
      <li
        class="link-bar-link"
      >
        Archive
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
